wgpu-hal/src/gles/adapter.rs

```diff
@@ -499,6 +499,16 @@ impl super::Adapter {
             0
         };
 
+        let max_sampled_textures_per_shader_stage = super::MAX_TEXTURE_SLOTS as u32;
+        let max_samplers_per_shader_stage = super::MAX_SAMPLERS as u32;
+
+        let max_bindings_per_bind_group = (max_sampled_textures_per_shader_stage
+            + max_samplers_per_shader_stage
+            + max_storage_buffers_per_shader_stage
+            + max_storage_textures_per_shader_stage
+            + max_uniform_buffers_per_shader_stage)
+            * crate::auxil::MAX_SHADER_STAGES_PER_PIPELINE;
+
```

Comment:
suggestion: This calculation is currently duplicated across all back ends. We should make this shared code, too.